integration-test/src/test/java/org/uniprot/store/indexer/search/uniprot/ValidateAllFieldsExistIT.java

```diff
@@ -0,0 +1,58 @@
+package org.uniprot.store.indexer.search.uniprot;
```

Comment:
As chatted about, yes this will be there in the abstract test layer, i have it as another jira where I'm trying to add tests at different layers.

This test checks that the schema is aligned with the enum -- so that whenever we use the enum, we are guaranteed it is valid.